cypress/integration/specialFeatures/utilities/scriptSwitchingJourneyAssertions/index.js

```diff
@@ -0,0 +1,56 @@
+import appConfig from '../../../../../src/server/utilities/serviceConfigs/index';
+
+const assertScriptCookie = (product, cookieValue) => {
+  const cookieSuffix = ['ukchina', 'zhongwen'].includes(product)
+    ? 'chinese'
+    : product;
+
+  cy.getCookie(`ckps_${cookieSuffix}`).should(
+    'have.property',
+    'value',
+    cookieValue,
+  );
+};
```

Comment:
Let's remove all references to cookies if we're not going to check them

Reply:
In progress k dog